beta/adobe-journey-optimizer/push-messaging-guide.md

```diff
@@ -0,0 +1,81 @@
+
+Mobile SDK has now integrated with [Project Griffon](https://aep-sdks.gitbook.io/docs/beta/project-griffon), which is a new, innovative beta product from Adobe Experience Cloud to help you inspect, proof, simulate, and validate how you collect data or serve experiences in your mobile app. With respect to push notifications, this integration helps validate various configurations across components involved in the data flow, by running a [test end-to-end push notification flow](https://aep-sdks.gitbook.io/docs/resources/troubleshooting-guides/troubleshooting-push). It shows errors encountered and recommendations to solve misconfigurations.
+
+#### Adobe Launch
+
+[Adobe Launch or Data Collection](https://experienceleague.adobe.com/docs/launch/using/home.html) is the next generation of mobile SDK management capabilities which enables data flow from mobile SDK to Experience Platform. It provides capabilities to register extensions, create rules and data elements to send data from your mobile app to the Adobe experience cloud solutions. With respect to the push notifications data flow, the primary configurations required in Launch are :
+
+- Creating a datastream to configure the profile and experience event datasets against which the data flows into experience platform. 
+- Creating a client-side mobile property and adding extensions. Mobile SDK closely integrates with these extensions to provide a seamless data collection experience.
+- Registering the mobile app bundle identifier and app credentials that would help to uniquely identify and validate sanity of the app at the time of delivering push notification. 
+
+You will find detailed steps for these configurations [here](https://aep-sdks.gitbook.io/docs/beta/adobe-journey-optimizer).
+
+#### Real-time Customer Profile
+
+[Real-time Customer Profile](https://experienceleague.adobe.com/docs/experience-platform/profile/home.html) is the component in [Experience Platform](https://experienceleague.adobe.com/docs/experience-platform.html) that allows you to see a holistic view of each individual customer by combining data from multiple channels, including web, mobile, CRM, and third party. Profile allows you to consolidate your customer data into a unified view offering an actionable, timestamped account of every customer interaction. 
+
+Static data that identifies the user of the mobile app such as a push token, is stored against the user's profile as record data while the interactions the user does with push notifications are tracked as time-series events data. 
```

Comment:
"... the mobile app, such as a push token, is stored against the user's profile as record data, while the user's interactions with push notifications are tracked as time-series event data."